app/screens/TalkDetailsScreen/TalkDetailsScreen.tsx

```diff
@@ -63,40 +148,36 @@ export const TalkDetailsScreen: FC<StackScreenProps<AppStackParamList, "TalkDeta
           </View>
           <View style={$containerSpacing}>
             <Image
-              source={isWorkshop ? workshopCurve : talkCurve}
-              style={isWorkshop ? $workshopCurve : $talkCurve}
+              source={variant === "workshop" ? workshopCurve : talkCurve}
```

Comment:
Nit: Maybe eliminate some duplication by re-purposing the `isWorkshop` variable as `const isWorkshop = variant === "workshop"`?